### PR TITLE
Maintenance: Replace constructed message with complete strings

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Based on a discussion in https://github.com/xbmc/Official-Kodi-Remote-iOS/pull/559 this PR replaces a constructed message with the complete string. This resolves issues with the localization for several languages. The translations were checked and adapted with help of Google Translator.

## Summary for release notes

Maintenance: Replace constructed message with complete strings

### CODE REVIEW

File: XBMC Remote/NowPlaying.m [hunk 1]
```diff
@@ -1791,17 +1791,16 @@ - (IBAction)buttonToggleItemInfo:(id)sender {
 
 - (void)showClearPlaylistAlert {
     if (!playlistView.hidden && self.view.superview != nil) {
-        NSString *playlistName = @"";
+        NSString *message = LOCALIZED_STR(@"Are you sure you want to clear the playlist?");
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:1794]: 1. please use `switch`
2. assign default value in the `default` case

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]
please also fix conflicts

File: XBMC Remote/NowPlaying.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m]: Done

**Review by kambala-decapitator — APPROVED** [review]

**Comment by wutschel** [comment]
Squashed and rebased to master.

**Pull request merged**